src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -190,10 +190,12 @@ public boolean isDependentArgumentAllowed(final Class<?> dependentClass) {
     // Return the allowable values for readFilterNames/disableReadFilter
     @Override
     public Set<String> getAllowedValuesForDescriptorArgument(final String longArgName) {
-        if (longArgName.equals(StandardArgumentDefinitions.READ_FILTER_LONG_NAME) ||
-                longArgName.equals(StandardArgumentDefinitions.DISABLE_READ_FILTER_LONG_NAME)) {
+        if (longArgName.equals(StandardArgumentDefinitions.READ_FILTER_LONG_NAME)) {
             return readFilters.keySet();
         }
+        if (longArgName.equals(StandardArgumentDefinitions.DISABLE_READ_FILTER_LONG_NAME)) {
+            return toolDefaultReadFilters.keySet();
```

Comment:
Last commit adds a test for this. Disabling a filter that is not enabled just log a warning. Anyway, the tool display in the help that the "allowed values" are the default ones. This is something quite nice to do not output all the filters twice in the help (both in `--readFilter` and `--disableReadFilter`) and give to the user the default filters that are enabled to make clear what can be disabled without a warning.